doc/MENTOR_GUIDELINES.md

```diff
@@ -0,0 +1,41 @@
+### How to be a GCI mentor
+
+To be a GCI mentor at Public labs, there are some prerequisites that we expect you to follow before we can invite you to be a mentor :
+
+1. Solve a [first-timers issue](https://code.publiclab.org/). Provide the link to the first timer issue's pull request (PR) once it's merged.
+
+2. Solve a [help-wanted issue](https://github.com/publiclab/plots2/labels/help-wanted). Help wanted issues are issues which are not labelled as `gci-candidate` neither as `fto-candidate` nor as `first-timer-only`. Provide link to such issues' merged PR.
+
+**(Make sure you claim the issue first by commenting on issue you are planning to solve). This is very important.**
+
+3. Create a **first-timers issue.** [Use the extra friendly template](https://publiclab.org/notes/warren/10-31-2016/create-a-welcoming-first-timers-only-issue-to-invite-new-software-contributors) which we generally use for creating our first timer issues. Provide the link to the first timer issue which is created by you.
+
+4. Once done with all these, fill [this](https://docs.google.com/forms/d/e/1FAIpQLScSBS-ddZN2H-lviUJbKlbV2VwP21fdIutXOCBzigRhmXJybw/viewform?usp=sf_link) form.
+<hr>
+
+* It's important to understand that people need to do only a single `first-timer-only` issue.
+
+* A first-timers-issue is meant to welcome people to open-source and should be easy enough to understand to make them feel comfortable. Use the [extra-friendly template](https://publiclab.org/n/13667) when creating `first-timer` issues.
+
+* Once done with that, they need to move on to `help-wanted` issues. **As a mentor, you need to assist them in these areas.**
+
+* A `help-wanted` issue is a bit more hard to tackle and people can move onto this once they find themselves comfortable by solving one `first-timer` issue. 
+
+* As a mentor, you also need to guide students to create good `first-timer` issues and help them in reviewing PRs.
+
+* We want to move towards a self-sustainable system where people make and review the PRs amongst themselves.
+
+* If you are creating a `gci-candidate issue` format it properly with
+lines like
+
+**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out [https://code.publiclab.org](https://code.publiclab.org) to see more.**
+
+* When creating an issue for GCI, you need to list the potential solutions too.
+
+* A GCI issue needs to be put in the [staging list](https://github.com/publiclab/plots2/issues/3276) as well.
+
+* You also need to mention the category of the issue(Code, Design, Documentation/training, Quality Assurance, Outreach) and tell the instance count, i.e., how many people can take that issue in the issue description.
+
+* For GCI issue you need to create the same task on the GCI dashboard as well.
+
+* You need to use extra-friendly tone at Public Lab even if people are not able to commit to send a PR on time.
```

Comment:
Can you please change this line? I was just explaining this to you. We can't write that people will not be able to complete issues in the documentation